File: 1580-ShuffleTheArray/1580-ShuffleTheArray.java
// Last updated: 3/26/2026, 1:48:19 AM
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] result = new int[nums.length];
        int k=0;
        for(int i=0;i+n<nums.length;i++)
        {
            result[k++]= nums[i];
            result[k++] = nums[i+n];
        }
        
       
        return result;
    }
}

File: 1580-ShuffleTheArray/1580-ShuffleTheArray.cs
// Last updated: 3/26/2026, 1:48:17 AM
public class Solution {
    public int[] Shuffle(int[] nums, int n) {
        int[] result = new int[nums.Length];
        int k=0;
        for(int i=0;i+n<nums.Length;i++)
        {
            result[k++]= nums[i];
            result[k++] = nums[i+n];
        }
        
       
        return result;
    }
}